packages/components/src/logos/cloud-logo/index.tsx

```diff
@@ -1,4 +1,4 @@
-const CloudLogo = ( props: React.SVGProps< SVGSVGElement > ) => (
+export const CloudLogo = ( props: React.SVGProps< SVGSVGElement > ) => (
```

Comment:
I was surprised we call this just "CloudLogo". The product is called "WP Cloud", so I'd expect the logo component to be called "WPCloudLogo" to reflect that.

Could be done in a follow-up if you like.